pkg/admission/chain.go

```diff
@@ -39,6 +39,9 @@ func NewFromPlugins(client client.Interface, pluginNames []string, configFilePat
 // Admit performs an admission control check using a chain of handlers, and returns immediately on first error
 func (admissionHandler chainAdmissionHandler) Admit(a Attributes) error {
 	for _, handler := range admissionHandler {
+		if !handler.Handles(a.GetOperation()) {
```

Comment:
I know we don't have one now, but we should have one.  Please add a test for the chainAdmissionHandler that takes a set of mock admission handlers, and ensure that it respects the handles method for the objects it delegates.